doc/classes/Image.xml

```diff
@@ -760,7 +760,39 @@
 		<constant name="FORMAT_ASTC_8x8_HDR" value="38" enum="Format">
 			Same format as [constant FORMAT_ASTC_8x8], but with the hint to let the GPU know it is used for HDR.
 		</constant>
-		<constant name="FORMAT_MAX" value="39" enum="Format">
+		<constant name="FORMAT_R16" value="39" enum="Format">
+			OpenGL texture format [code]GL_R16[/code] where there's one component, a 16-bit unsigned normalized integer value.
+			[b]Note:[/b] due to limited hardware support, it is mainly recommended to be used on desktop or console devices. It may be unsupported on mobile or web, and will consequently be converted to [constant FORMAT_RF].
```

Comment:
```suggestion
			[b]Note:[/b] Due to limited hardware support, it is mainly recommended to be used on desktop or console devices. It may be unsupported on mobile or web, and will consequently be converted to [constant FORMAT_RF].
```

Same for the other notes.